examples/maps-named/named.js

```diff
@@ -0,0 +1,96 @@
+var CartoDB = require('../..');
```

Comment:
Sry, thought it was all commented out.
You can safely remove comments. There's always a way to retrieve history with git.